projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,6 +18,59 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const ConnectionItem = props => {
+	const { connections } = window.jetpackSocialInitialState;
+
+	return props.connectionIds.map( connectionId => {
+		const connection = connections[ props.provider ][ connectionId ];
+		return (
+			<div
+				key={ connectionId + Date.now().toString( 36 ) + Math.random().toString( 36 ).substr( 2 ) }
+			>
+				<tr>
```

Comment:
Should we just have the header row once, and then we could have a row for each connection? It would also mean that we wouldn't have to wrap this in a `div`

Reply:
This might make me change the CSS a little bit and that broke the UI. Also not differentiate b/w the different providers, 

Is it OK if we let it be for now and address it in beta?

Reply:
Sure, I think we will want to play with the layout/HTML as we got some designs for this.